added share bar ribbon

bhansa opened the pull request:
Fixes #86
added vertical share bar for desktop only with sharer links.

bhansa commented:
#86

bhansa commented:
![image](https://user-images.githubusercontent.com/9723884/40281317-f0aaa9b0-5c7d-11e8-905a-77f0a311edd9.png)

temporary preview URL: http://947b20e7.ngrok.io/

vipulgupta2048 commented:
Thanks for that, it might look even better if the box is transparent and on the left side.

bhansa commented:
![image](https://user-images.githubusercontent.com/9723884/40282088-4fbae18c-5c87-11e8-82f8-8bdf20beca4c.png)
Updated the alignment, made it a little transparent and added transition(scale) for icons on hover, since opacity doesn't work well on transparent background.

vipulgupta2048 commented:
Both look demos look brilliant, I guess. @pythonindia/pycon-2018-team please review.

ananyo2012 commented on index.html:
```diff
@@ -95,6 +95,19 @@
+         <ul class="list-vertical">
+            <li class="list-vertical__label">Share</li>
+
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image" class="list-vertical__icon"></a></li>
```
Can we make these links a share button instead of just page links ?

bhansa replied:
I mentioned the same in the pull request as well, since I had a look at pycon2017 website which has the same functionality. If someone can provide me the sharer links, I can add them here.

ananyo2012 replied:
I used http://www.sharelinkgenerator.com/ for generating the links. In case you want to change the content just head over there to generate the links with your content.

bhansa replied:
thanks @ananyo2012, I'll update them today

bhansa replied:
looks like we can only share URL with the Facebook post, and it needs a valid URL.
I have updated the URL for it.

bhansa replied:
Also, what should be a better subject line for email share. I am keeping it *"PyCon India 2018 - Hyderabad"*. 
Let me know your suggestions.

ananyo2012 commented:
Merging this in

vipulgupta2048 commented:
Hi, I almost forget about this PR and seeing the pushed changes, I would like to propose something. By transparent, I would like to suggest this @pythonindia/pycon-2018-team 
![slfjds](https://user-images.githubusercontent.com/22801822/40703677-854f282a-6403-11e8-8781-019882d5bd13.jpg)

Since reverting is a pain, I would like to open a new PR. All suggestions are appreciated.

vipulgupta2048 commented:
I was just suggesting.

ananyo2012 commented:
Well I am not an UX guy either. @rajataaron Can you comment here ?

rajataaron commented:
I agreed to @ananyo2012 I also like the translucent background and just use the color of background rgb(84, 50, 120). And I have seen it on website I think it should be fixed so user can use share button even if he/she at bottom. So use position:fixed and also give it a z-index value so it won't get affected by other elements.

Sharing the screenshot here for reference.
![screenshot-2018-5-31 pycon india 2018 in hyderabad october 5th to 9th](https://user-images.githubusercontent.com/7890954/40767367-f6ab66a6-64cf-11e8-8db0-fcadaf9b248a.png)
![screenshot-2018-5-31 pycon india 2018 in hyderabad october 5th to 9th 1](https://user-images.githubusercontent.com/7890954/40767369-f6dc5be4-64cf-11e8-97c9-13335714b248.png)

ananyo2012 commented:
@bhansa Can you take care of the comments Rajat mentioned ? Please Open a
new PR with the changes.

On Thu, May 31, 2018 at 12:41 PM Rajat Saini <notifications@github.com>
wrote:

> I agreed to @ananyo2012 <https://github.com/ananyo2012> I also like the
> translucent background and just use the color of background rgb(84, 50,
> 120). And I have seen it on website I think it should be fixed so user can
> use share button even if he/she at bottom. So use position:fixed and also
> give it a z-index value so it won't get affected by other elements.
>
> Sharing the screenshot here for reference.
> [image: screenshot-2018-5-31 pycon india 2018 in hyderabad october 5th to
> 9th]
> <https://user-images.githubusercontent.com/7890954/40767367-f6ab66a6-64cf-11e8-8db0-fcadaf9b248a.png>
> [image: screenshot-2018-5-31 pycon india 2018 in hyderabad october 5th to
> 9th 1]
> <https://user-images.githubusercontent.com/7890954/40767369-f6dc5be4-64cf-11e8-97c9-13335714b248.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pythonindia/inpycon2018/pull/89#issuecomment-393431895>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKACRwWAzGvgJW19rdo6355IDdW7e1Szks5t35e0gaJpZM4UGCcO>
> .
>

bhansa commented:
sure @ananyo2012 , I have changed it already will share the preview with you guys.
